Author: Jamie Strandboge <jamie.strandboge@ubuntu.com>
Description: Adjust build system to build what we want
 The upstream build system doesn't have particularly good support for
 configuring our target via commandline options, so we instead patch the
 generated files.
Index: edk2-0~20121205.edae8d2d/Conf/target.txt
===================================================================
--- edk2-0~20121205.edae8d2d.orig/Conf/target.txt
+++ edk2-0~20121205.edae8d2d/Conf/target.txt
@@ -23,7 +23,9 @@
 #                                               build. This line is required if and only if the current
 #                                               working directory does not contain one or more description
 #                                               files.
-ACTIVE_PLATFORM       = Nt32Pkg/Nt32Pkg.dsc
+#ACTIVE_PLATFORM       = MdeModulePkg/MdeModulePkg.dsc
+ACTIVE_PLATFORM       = OvmfPkg/OvmfPkgX64.dsc
+ 
 
 #  TARGET                List       Optional    Zero or more of the following: DEBUG, RELEASE, NOOPT
 #                                               UserDefined; separated by a space character.
@@ -46,7 +48,7 @@
 #                                               If not specified, then all valid architectures specified
 #                                               in the platform file, for which tools are available, will be
 #                                               built.
-TARGET_ARCH           = IA32
+TARGET_ARCH           = X64
 
 #  TOOL_DEFINITION_FILE  Filename  Optional   Specify the name of the filename to use for specifying
 #                                             the tools to use for the build.  If not specified,
@@ -56,12 +58,12 @@
 #  TAGNAME               List      Optional   Specify the name(s) of the tools_def.txt TagName to use.
 #                                             If not specified, all applicable TagName tools will be
 #                                             used for the build.  The list uses space character separation.
-TOOL_CHAIN_TAG        = MYTOOLS
+TOOL_CHAIN_TAG        = GCC49
 
 # MAX_CONCURRENT_THREAD_NUMBER  NUMBER  Optional  The number of concurrent threads. Recommend to set this
 #                                                 value to one more than the number of your compurter
 #                                                 cores or CPUs. Less than 2 means disable multithread build.
-MAX_CONCURRENT_THREAD_NUMBER = 1
+MAX_CONCURRENT_THREAD_NUMBER = 5
 
 
 # BUILD_RULE_CONF  Filename Optional  Specify the file name to use for the build rules that are followed
